classes/person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(name, age = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def of_age?
+    age > 18
+  end
+
+  def can_use_services?
+    parent_permission == true || age >= 18
```

Comment:
- [ ] You are duplicating the code at this line, you have a method that returns true if the age is over 18, why don't you use it in the statement instead of redefining age > 18? please make sure you fix it